net.rocketpool/image/src/app.ts

```diff
@@ -19,145 +23,118 @@ interface Event {
   logIndex: number;
 }
 
+type Provider = ethers.providers.JsonRpcProvider | ethers.providers.WebSocketProvider;
+
 /////////////////////////////////////////////////////////////////////////////////////////
 
 async function readEventsFromETH(
-  startBlock,
-  endBlock,
-  clb: (_: Event) => boolean,
-  stride = 10000
-) {
+  rethContract: RethAbi,
+  startBlock: number,
+  endBlock: number,
+  clb: (_: EventRow) => boolean,
+): Promise<void> {
   console.log(
-    "Considering events in block range [%s, %s] (%s blocks)",
-    startBlock,
-    endBlock,
-    endBlock - startBlock
+    `Considering events in block range [${startBlock}, ${endBlock}] (${endBlock - startBlock})`,
```

Comment:
I think we should return the "(N blocks)" part as it will be unclear what the number in round brackets represents

Reply:
Done